pkg/scaffold/controller_kind.go

```diff
@@ -44,7 +44,7 @@ const controllerKindTemplate = `package {{ .Resource.LowerKind }}
 import (
 	"context"
 
-	{{ .Resource.Group}}{{ .Resource.Version }} "{{ .Repo }}/pkg/apis/{{ .Resource.Group}}/{{ .Resource.Version }}"
+	{{ (rmhyphen .Resource.Group)}}{{ .Resource.Version }} "{{ .Repo }}/pkg/apis/{{ .Resource.Group}}/{{ .Resource.Version }}"
```

Comment:
We will still end up with a hyphen in the package import path with this because we're only removing the hyphen from the import alias, right?

Perhaps we should add a new field to the resource called `GoImportGroup` that follows the go import path and package naming convention. Then we can just use that everywhere we reference the import path rather than using `rmhyphen`

We would also need to use the `GoImportGroup` field when scaffolding the `apis` subdirectories for this GVK.

@nak @LiliC @estroz Thoughts?